packages/accounts/src/msca/plugins/multisig/plugin.ts

```diff
@@ -154,7 +154,16 @@ export type MultisigPluginActions<
   ReadAndEncodeActions<TAccount>;
 
 const addresses = {
-  11155111: "0x0b135A12EB2f7b441DCcE5F7DE07DB65AE7c4649" as Address,
+  1: "0x000000000000A53f64b7bcf4Cd59624943C43Fc7" as Address,
```

Comment:
nit you can use [<chain>.id] here.

Reply:
TIL. this is autogenerated by plugingen though, probably best to leave it as is

Reply:
could i do that change above in config.ts?

Reply:
no this is fine. auto-geneated file.